plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -28,44 +34,41 @@ import {
   TableOptions,
 } from 'react-table';
 import { Row, Col } from 'react-bootstrap';
-import {
-  FaSort,
-  FaSortUp, // asc
-  FaSortDown, // desc
-} from 'react-icons/fa';
 import GlobalFilter from './components/GlobalFilter';
-import SelectPageSize from './components/SelectPageSize';
+import SelectPageSize, { SizeOption } from './components/SelectPageSize';
 import SimplePagination from './components/Pagination';
-import Styles from './Styles';
 import useSticky from './hooks/useSticky';
 import useColumnCellProps from './hooks/useColumnCellProps';
 
-export interface DataTableProps<D extends object>
-  extends Omit<TableOptions<D>, 'getStickyTableSize'> {
-  className?: string;
+export interface DataTableProps<D extends object> extends Omit<TableOptions<D>, 'getTableSize'> {
+  tableClassName?: string;
   showSearchInput?: boolean;
-  pageSizeOptions?: number[]; // available page size options
+  pageSizeOptions?: SizeOption[]; // available page size options
+  maxPageItemCount?: number;
   hooks?: PluginHook<D>[]; // any additional hooks
   width?: string | number;
   height?: string | number;
   pageSize?: number;
+  noResults?: string | ((filterString: string) => ReactNode);
```

Comment:
`noResultsNode` or `noResultsMessage`? (this reads like a `boolean` to me)

Reply:
Will use `noResultsMessage`.